zerver/forms.py

```diff
@@ -224,12 +225,58 @@ def validate(self, emails):
 
 class FindMyTeamForm(forms.Form):
     emails = MultiEmailField(
-        help_text="Add up to 10 comma-separated email addresses.")
+        help_text=_("Add up to 10 comma-separated email addresses."))
 
     def clean_emails(self):
         # type: () -> List[Text]
         emails = self.cleaned_data['emails']
         if len(emails) > 10:
-            raise forms.ValidationError("Please enter at most 10 emails.")
+            raise forms.ValidationError(_("Please enter at most 10 emails."))
 
         return emails
+
+class ProfileFieldForm(forms.ModelForm):
+    class Meta(object):
+        model = CustomProfileField
+        fields = ['name', 'field_type']
+
+def validate_custom_field_value(field, value):
+    # type: (CustomProfileField, Text) -> None
+    field_type = field.field_type
+    name = field.name
+    field_type_error = _("{name} does not have {field_type} type.")
+    if field_type == CustomProfileField.INTEGER:
+        try:
+            int(value)
+        except ValueError:
+            raise ValidationError(
+                field_type_error.format(name=name, field_type=field_type))
+    elif field_type == CustomProfileField.DOUBLE:
+        try:
+            float(value)
+        except ValueError:
+            raise ValidationError(
+                field_type_error.format(name=name, field_type=field_type))
+    elif field_type == CustomProfileField.SHORT_TEXT:
+        max_length = 200
```

Comment:
let's move this constant to the models file as CustomProfileField.MAX_SHORT_TEXT_LENGTH, so that we can use the same length constant everywhere.